core/runtime/binaryen/runtime_api/grandpa_impl.cpp

```diff
@@ -16,8 +16,9 @@ namespace kagome::runtime::binaryen {
   using primitives::SessionKey;
 
   GrandpaImpl::GrandpaImpl(
+      std::shared_ptr<WasmProvider> wasm_provider,
       const std::shared_ptr<RuntimeManager> &runtime_manager)
-      : RuntimeApi(runtime_manager) {}
+      : RuntimeApi(std::move(wasm_provider), runtime_manager) {}
```

Comment:
May be better use const-ref to avoid make temp-object

Reply:
Are you sure there will be one?

Reply:
Absulutly.